code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -89,31 +89,48 @@
 	Can feathers gain their own wings?"
 	icon_state = "solemnlament"
 	inhand_icon_state = "solemnlament"
+	special = "Firing both solemn lament and solemn vow at the same time will increase damage by 1.5x"
 	ammo_type = /obj/item/ammo_casing/caseless/ego_solemnlament
 	burst_size = 1
 	fire_delay = 0
 	fire_sound = 'sound/abnormalities/funeral/spiritgunwhite.ogg'
 	fire_sound_volume = 30
+	attribute_requirements = list(JUSTICE_ATTRIBUTE = 60)
+
+/obj/item/gun/ego_gun/pistol/solemnlament/before_firing(atom/target, mob/user)
+	for(var/obj/item/gun/ego_gun/pistol/solemnvow/Vow in user.held_items)
```

Comment:
Might be a better idea to add break here, or use locate().

Reply:
I can break